Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Multistage docker build, requires docker 17.05
 
 # builder stage
-FROM rust:1.26 as builder
+FROM rust:1.29.0 as builder
```

Comment:
It looks like the official Rust image now include the patch version. There's no tag for `1.29`. 😢